cli_tools/gce_export/main.go

```diff
@@ -94,15 +95,42 @@ func main() {
 		log.Fatal(err)
 	}
 
-	ctx := context.Background()
-	client, err := storage.NewClient(ctx, option.WithServiceAccountFile(*oauth))
-	if err != nil {
-		log.Fatal(err)
+	writer, targetPath := createWriter()
+	createGzipFile(file, size, writer, targetPath)
+}
+
+func createWriter() (io.WriteCloser, string) {
+	var w io.WriteCloser
+	var p string
+	if *gcsPath != "" {
+		bkt, obj, err := storageutils.SplitGCSPath(*gcsPath)
+		if err != nil {
```

Comment:
It's not a good practice to perform fatal on an app in a method outside main. Return error and let main decide if it can recover or not. The same applies for other instances of log.fatal in sub-methods.